rfcs/text/0042-risk-score-extensions.md

```diff
@@ -24,16 +21,50 @@ This RFC seeks to extend the [existing risk fields](https://www.elastic.co/guide
     * These categories' definitions may be _extended_ in the future to include new data sources
   * Due to the above category traits, we need to come up with a naming convention for these categorical score fields that allows them to be extended without invalidating the existing field names.
 
-<!--
-Stage 1: If the changes include field additions or modifications, please create a folder titled as the RFC number under rfcs/text/. This will be where proposed schema changes as standalone YAML files or extended example mappings and larger source documents will go as the RFC is iterated upon.
--->
 
 <!--
 Stage X: Provide a brief explanation of why the proposal is being marked as abandoned. This is useful context for anyone revisiting this proposal or considering similar changes later on.
 -->
 
 ## Fields
 
+I'll enumerate the fields being introduced here grouped by their motivation/goal:
+
+### Identifier Fields
+These fields are intended to allow future extensibility of our concept of an "identifier." Currently, we leverage two such fields: `host.name` and `user.name`, to group documents for the purposes of risk scoring, but we definitely imagine this being extended in the future to allow multiple such fields for hosts and users, or more simply to allow administrators to configure fields other than those above. The presence of these two fields allows us to audit/explain how a particular risk score was identified.
+
+* `id_field`
+* `id_value`
+
+
+### Risk Category Fields
+Some of the context here was discussed in Stage 0; please read the above for that. More specifically, these fields seek to provide the category contributions to the score, and the number of risk inputs in that category, across each of the five proposed categories:
+
+* `category_1_score`
```

Comment:
How do users know what Category 1, 2...etc. mean? Would it make sense for these field names to reflect what the category means, for example `alert_category_score`, `posture_category_count` etc?

Reply:
I was thinking that too, but it seems that similar to `id_field` and whatnot, ideally the content of each numbered category could be changed without changing the fields? e.g. `category_1` could be from alerts one day, and changed to something else in the future 🤔

Reply:
Great question! The more specific `alert_category_score` etc. were actually the original proposition, but we decided to go the more generic route for future-proofing. 

@susan-shu-c got the gist of it, although more accurately we want to allow _extending_ each category's definition as opposed to outright changing it (for backwards compatibility, etc.). For example, `category_1` currently includes alerts, but might be extended in the future to allow more alert-like sources. 

This all was described in a little more detail in [stage 0](https://github.com/elastic/ecs/pull/2232/files#diff-40e16fa32db2ffbdeabf56eb2ffeac75dfa12410e662c3328c7c859b426805d2R20-R25) if you're interested 👍